### ISSUE TITLE
When running more than one project, since 1.7.1, need to ddev remove, ddev config, ddev import-db each time

### ISSUE THREAD

**Issue by scottsoapbox** [drud/ddev#1578]
When running more than one project, since 1.7.1, need to `ddev remove`, `ddev config`, `ddev import-db` each time. Previously, could just run `ddev start`, but now doing that results in the below error.

```
ddev start
Starting align-platform... 
Running Command  Command=ip address show dev docker0
Recreating ddev-align-platform-db ... error
 
ERROR: for ddev-align-platform-db  Cannot create container for service db: Duplicate mount point: /var/lib/mysql 
 
ERROR: for db  Cannot create container for service db: Duplicate mount point: /var/lib/mysql 
Encountered errors while bringing up the project. 
Failed to start align-platform: Failed to run docker-compose [-f /home/scott/www/align-platform/.ddev/docker-compose.yaml up -d], err='exit status 1', stdout='', stderr='RRecreating ddev-align-platform-db ... error

ERROR: for ddev-align-platform-db  Cannot create container for service db: Duplicate mount point: /var/lib/mysql

ERROR: for db  Cannot create container for service db: Duplicate mount point: /var/lib/mysql
Encountered errors while bringing up the project.' 
```

Config file
```
APIVersion: v1.7.1
name: align-platform
type: drupal8
docroot: web
php_version: "7.2"
webserver_type: nginx-fpm
router_http_port: "80"
router_https_port: "443"
xdebug_enabled: false
additional_hostnames: []
additional_fqdns: []
mariadb_version: "10.2"
webcache_enabled: false
nfs_mount_enabled: false
provider: default
```

OS information:
Ubuntu 18.04
ddev version
```
bgsync        	drud/ddev-bgsync:v1.7.0       
cli           	v1.7.1                        
commit        	v1.7.1                        
db            	drud/ddev-dbserver:v1.7.0-10.2
dba           	drud/phpmyadmin:v1.7.0        
ddev-ssh-agent	drud/ddev-ssh-agent:v1.7.0    
docker        	18.09.6                       
docker-compose	1.22.0                        
domain        	ddev.local                    
router        	drud/ddev-router:v1.7.0       
web           	drud/ddev-webserver:v1.7.0 
```

Hope someone is able to help!

**Comment by rfay** [comment]
Thanks so much for your report. I take it really seriously; somebody else reported similar behavior (apparent shared db volume), but I'm completely unable to recreate it, and know of no recent changes that would affect it.

* Is it possible that you have two projects named "align-platform" (looking at the 'name' entry in config.yaml? Names of projects must definitely be unique. Please do provide the config.yaml for the *both* projects that you're trying to run at the same time.
* If you could, please try creating two very simple projects, perhaps named test1 and test2, just `ddev config` them and try to run them at the same time. I think you'll find they work ok. 
* Please provide the output of `docker volume ls | grep mariadb`
* Could you please provide (probably as a gist or other link) the output of `docker inspect --format='{{json .Mounts}}'  ddev-align-platform-db`?  You'll see the mount and the source. If the source is the same between the two projects, we definitely have a problem.

Thanks for your help on this, I sure hope we can nail it.

**Comment by rfay** [comment]
Oh, are you completely sure that you have to import-db to solve the problem? Perhaps you were just using "start" and "stop" and have the problem described in https://github.com/drud/ddev/issues/1541 ? In that case, `ddev rm` and `ddev start` will work just fine, but `ddev stop` and `ddev start` (or `ddev restart`) result in a problem like yours. This behavior is fixed in upcoming v1.8.0.

**Comment by scottsoapbox** [comment]
Ah great! I wrongly assumed `ddev rm` would remove the database since the container was removed. Will do `ddev rm && ddev start` when we encounter this issue for now then. Thank you for the quick reply!

**Comment by alesrebec** [comment]
I'm also experiencing this with 1.7.1
Wasn't sure what the problem was but suddenly all project started reporting this (I use ddev for multiple projects at once)

```
ERROR: for 57ebb448590a_PROJECT-db  Cannot create container for service db: Duplicate mount point: /var/lib/mysql

ERROR: for db  Cannot create container for service db: Duplicate mount point: /var/lib/mysql
```

Before 1.7.1 I wasn't experiencing this.
Will try with v1.8.0-rc1 if this is still the case.

**Comment by rfay** [comment]
@alesrebec does `ddev rm && ddev start` fix this for you? I'd sure love to understand what's really going on here.

**Comment by alesrebec** [comment]
Will try the next time it happens to me.

**Comment by alesrebec** [comment]
Ok, I stayed on 1.7.1 for that reason.

I know how to reproduce it now.
If I stop the project with "ddev stop" and then start it again with "ddev start" I get this error `Cannot create container for service db: Duplicate mount point: /var/lib/mysql`

Running `ddev rm && ddev start` does solve it and db data persists.

Probably this is solved already in 1.8.0 @rfay?

**Comment by rfay** [comment]
That problem was in fact solved in 1.8.0, assuming you have custom mysql settings, https://github.com/drud/ddev/issues/1541; anyway, pretty sure you won't see that any more in 1.8.0

**Comment by alesrebec** [comment]
I can confirm.
It's fixed in 1.8.0.

**Comment by rfay** [comment]
Thanks so much!
